docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -845,10 +852,30 @@ volume on the first run.
 
 The path of a device which is mounted for the volume.
 
+### `Driver=`
+
+Specify the volume driver name. When set to `image`, the `Image` key must also be set
+
+This is equivalent to the Podman `--driver` option
+
 ### `Group=`
 
 The host (numeric) GID, or group name to use as the group for the volume
 
+### `Image=`
+
+Specifies the image the volume is based on, when `Driver` is set the `image`.
+It is recommended to use a fully qualified image name rather than a short name, both for
+performance and robustness reasons.
+
+The format of the name is the same as when passed to `podman pull`, so it supports e.g., using
```

Comment:
soft suggestion, I'd drop the "e.g.,"

Reply:
I've rephrased the sentence (which I actually copied, so it appears three times now)